models/netsuite2/intermediate/int_netsuite2__acctxperiod_exchange_rate_map.sql

```diff
@@ -31,7 +31,9 @@ period_exchange_rate_map as ( -- exchange rates used, by accounting period, to c
   from consolidated_exchange_rates
 
   where consolidated_exchange_rates.to_subsidiary_id in (select subsidiary_id from subsidiaries where parent_id is null)  -- constrait - only the primary subsidiary has no parent
+  {% if var('netsuite2__using_accounting_book_subsidiaries', true) %}
     and consolidated_exchange_rates.accounting_book_id in (select accounting_book_id from accounting_books where is_primary)
```

Comment:
i'm unsure if just bypassing this where clause makes sense for the use case of not having accounting_book_subsidiaries enabled... thoughts @fivetran-joemarkiewicz

Reply:
ah yeah we talked about this offline